config/locales/de/controllers/collections.yml

```
@@ -19,6 +19,7 @@ de:
         already_member: Sie sind bereits ein Mitglied dieser Sammlung.
       success_notice: Sie können jetzt an dieser Sammlung zusammenarbeiten.
     share:
+      duplicate_share: Sie haben die Sammlung bereits mit diesem Nutzer geteilt.
```

I would suggest the following gender-neutral wording, which is in line with the remaining translations:

```suggestion
      duplicate_share: Sie haben die Sammlung bereits mit dieser/diesem Benutzer:in geteilt.
```

... or something different, if we move this to the `errors` (see other comment).